example/android/build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenLocal()
```

Comment:
I think `mavenLocal` is not necessary in this file. Is there a reason for adding it?

Reply:
You are right, removed mavenLocal()